packages/compressed-textures/src/const.ts

```diff
@@ -116,9 +116,9 @@ export enum INTERNAL_FORMATS
      */
     COMPRESSED_RGB_ATC_WEBGL = 0x8C92,
     /**
-     * @default 0x8C92
+     * @default 0x8C93
      */
-    COMPRESSED_RGBA_ATC_EXPLICIT_ALPHA_WEBGL = 0x8C92, // TODO: Probably a bug on the MDN site
+    COMPRESSED_RGBA_ATC_EXPLICIT_ALPHA_WEBGL = 0x8C93,
```

Comment:
Correct value in spec is here: https://registry.khronos.org/webgl/extensions/rejected/WEBGL_compressed_texture_atc/

MDN is likely wrong, e.g.: 
https://developer.mozilla.org/ja/docs/Web/API/WebGL_API/Constants#webgl_compressed_texture_atc